src/constants/index.js

```diff
@@ -26,7 +26,7 @@ export const Regex = {
   email: /^([a-zA-Z0-9])(([a-zA-Z0-9])*([._+-])*([a-zA-Z0-9]))*@(([a-zA-Z0-9-])+(\.))+([a-zA-Z]{2,4})+$/,
   phone: /^(\+\d{1,2}\s)?\(?\d{3}\)?[\s.-]?\d{3}[\s.-]?\d{4}$/,
   zip: /^(\d{5})$/,
-  password: /^([A-Za-z\d]{8,})$/,
+  password: /^([A-Za-z\d !"#$%&'()*+,-./:;<=>?@\\[\]^_`{|}~]{8,})$/,
```

Comment:
All of these including the space are literal special characters. The `\` is repeated twice, once to escape itself and later to escape `]`, which would otherwise have a meaning in the regex (for example, notice the last `]`, which was already included and is not escaped is what terminates the list of special characters in the regex.